bug/package.json

```diff
@@ -9,13 +9,15 @@
     "seed": "./node_modules/.bin/knex seed:run"
   },
   "dependencies": {
+    "camelcase": "^5.0.0",
     "config": "2.0.1",
     "express": "4.16.3",
     "js-yaml": "3.12.0",
     "knex": "0.15.2",
     "lodash": "^4.17.11",
     "moment": "^2.22.2",
-    "pg": "^7.4.3"
+    "pg": "^7.4.3",
```

Comment:
It will be good practice too, to use exact version e.g. `pg: 7.4.3` This will guarantee that dependencies will run on the exact same version.